### ISSUE TITLE
vstemplate project collection not included

### ISSUE THREAD

**Issue by robbypranata010118** [taori/Amusoft.VisualStudio.TemplateGenerator#22]
I am trying to generate my solution with a folder structure like this
![image](https://user-images.githubusercontent.com/70742769/92209591-76eeaa80-eeb7-11ea-9cfb-0a106530a19d.png)

and use a configuration like this
![image](https://user-images.githubusercontent.com/70742769/92209630-8a017a80-eeb7-11ea-83dc-87ad24809be0.png)
but the result is like this
![image](https://user-images.githubusercontent.com/70742769/92209655-94237900-eeb7-11ea-81a6-2ef9b1870bca.png)
my projects are not included in the vstemplate

but if I use a configuration like this
![image](https://user-images.githubusercontent.com/70742769/92209747-ba491900-eeb7-11ea-845e-a1bf55649296.png)
generator showing error and error detail like this
![image](https://user-images.githubusercontent.com/70742769/92209794-d056d980-eeb7-11ea-94ce-8ba3902d31e2.png)
can u give me some advice how to correct configuration to make the generator

**Comment by taori** [comment]
Can you upload a sample solution with the given structure? This looks like a bug to me. Also creating subfolders like you did in your later attempt is the correct approach for the structure you want to achieve.

**Comment by robbypranata010118** [comment]
`
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio Version 16
VisualStudioVersion = 16.0.30104.148
MinimumVisualStudioVersion = 10.0.40219.1
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Application", "Application", "{8EC25FD6-E271-4B1C-9CF7-DC2393B5BEE9}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MasterTemplate.Application", "Application\MasterTemplate.Application.csproj", "{E68A866A-2E5A-4A1E-BF87-72F31D4A5A7E}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Domain", "Domain", "{A407E5A4-8C29-46F6-B6DE-AB8AE94FB150}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MasterTemplate.Domain", "Domain\MasterTemplate.Domain.csproj", "{522D233E-28D3-4260-84A3-A1BE2CD767C7}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Infrastructure", "Infrastructure", "{35591CEE-CE61-4125-AD1D-9BE92F289672}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MasterTemplate.Infrastructure", "Infrastructure\MasterTemplate.Infrastructure.csproj", "{630BBD17-F65B-470A-AB10-D367901F2C3A}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Presentation", "Presentation", "{290EEB2A-8B51-4C7E-91EE-0E1E1ED9F294}"
EndProject
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "UnitTest", "UnitTest", "{D0BEDE1B-5C88-4740-A747-FC978B95C67F}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MasterTemplate.Presentation", "Presentation\MasterTemplate.Presentation.csproj", "{F4E61E46-A250-487F-889C-6E62DCE415FC}"
EndProject
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTest.Application", "UnitTest\UnitTest.Application\UnitTest.Application.csproj", "{95B8FC5C-9C9E-41AF-A401-FB19657A34D4}"
EndProject
Global
	GlobalSection(SolutionConfigurationPlatforms) = preSolution
		Debug|Any CPU = Debug|Any CPU
		Release|Any CPU = Release|Any CPU
	EndGlobalSection
	GlobalSection(ProjectConfigurationPlatforms) = postSolution
		{E68A866A-2E5A-4A1E-BF87-72F31D4A5A7E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{E68A866A-2E5A-4A1E-BF87-72F31D4A5A7E}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{E68A866A-2E5A-4A1E-BF87-72F31D4A5A7E}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{E68A866A-2E5A-4A1E-BF87-72F31D4A5A7E}.Release|Any CPU.Build.0 = Release|Any CPU
		{522D233E-28D3-4260-84A3-A1BE2CD767C7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{522D233E-28D3-4260-84A3-A1BE2CD767C7}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{522D233E-28D3-4260-84A3-A1BE2CD767C7}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{522D233E-28D3-4260-84A3-A1BE2CD767C7}.Release|Any CPU.Build.0 = Release|Any CPU
		{630BBD17-F65B-470A-AB10-D367901F2C3A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{630BBD17-F65B-470A-AB10-D367901F2C3A}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{630BBD17-F65B-470A-AB10-D367901F2C3A}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{630BBD17-F65B-470A-AB10-D367901F2C3A}.Release|Any CPU.Build.0 = Release|Any CPU
		{F4E61E46-A250-487F-889C-6E62DCE415FC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{F4E61E46-A250-487F-889C-6E62DCE415FC}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{F4E61E46-A250-487F-889C-6E62DCE415FC}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{F4E61E46-A250-487F-889C-6E62DCE415FC}.Release|Any CPU.Build.0 = Release|Any CPU
		{95B8FC5C-9C9E-41AF-A401-FB19657A34D4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
		{95B8FC5C-9C9E-41AF-A401-FB19657A34D4}.Debug|Any CPU.Build.0 = Debug|Any CPU
		{95B8FC5C-9C9E-41AF-A401-FB19657A34D4}.Release|Any CPU.ActiveCfg = Release|Any CPU
		{95B8FC5C-9C9E-41AF-A401-FB19657A34D4}.Release|Any CPU.Build.0 = Release|Any CPU
	EndGlobalSection
	GlobalSection(SolutionProperties) = preSolution
		HideSolutionNode = FALSE
	EndGlobalSection
	GlobalSection(NestedProjects) = preSolution
		{E68A866A-2E5A-4A1E-BF87-72F31D4A5A7E} = {8EC25FD6-E271-4B1C-9CF7-DC2393B5BEE9}
		{522D233E-28D3-4260-84A3-A1BE2CD767C7} = {A407E5A4-8C29-46F6-B6DE-AB8AE94FB150}
		{630BBD17-F65B-470A-AB10-D367901F2C3A} = {35591CEE-CE61-4125-AD1D-9BE92F289672}
		{F4E61E46-A250-487F-889C-6E62DCE415FC} = {290EEB2A-8B51-4C7E-91EE-0E1E1ED9F294}
		{95B8FC5C-9C9E-41AF-A401-FB19657A34D4} = {D0BEDE1B-5C88-4740-A747-FC978B95C67F}
	EndGlobalSection
	GlobalSection(ExtensibilityGlobals) = postSolution
		SolutionGuid = {55E23622-F82E-4494-8529-790949C73A7A}
	EndGlobalSection
EndGlobal
`

here my solution

**Comment by taori** [comment]
I meant a zipped solution, not the solution file. Once you provide me with that i can help you

**Comment by robbypranata010118** [comment]
Sorry For Late Response
Did You Mean This
[test2.zip](https://github.com/taori/Amusoft.VisualStudio.TemplateGenerator/files/5214601/test2.zip)

**Comment by taori** [comment]
@robbypranata010118 Not quite. What i am looking for is the solution that you are trying to generate a template from. That way i can run the config on my end and see what might be wrong with it.

**Comment by angelicochris** [comment]
I am having a similar problem here is my solution
http://www.mediafire.com/file/fjtb2a6g02amjf8/StandardMVC.zip/file
I also attached my exported config file

[StandardMVC.zip](https://github.com/taori/Amusoft.VisualStudio.TemplateGenerator/files/5990008/StandardMVC.zip)

**Comment by taori** [comment]
@angelicochris Thanks for the sample. i will look into it
